js/script.js

```diff
@@ -118,12 +118,12 @@ function setGamePoints() {
 
 function checkWin() {
     if (player.score == 10) {
-        alert(player.name + 'wins');
+        alert(player.name + ' wins');
         gameState = 'ended';
+        setGameElements('ended');
     } else if (computer.score == 10) {
         alert("Computer wins");
         gameState = 'ended';
+        setGameElements('ended');
```

Comment:
Unnecessary, sorry for that. I was test something and forget to remove it :) Today (8.06, because is after midnight) I will remove it and send to check.